src/org/redvsdead/webserver/util/FileUtils.java

```diff
@@ -0,0 +1,15 @@
+package org.redvsdead.webserver.util;
+
+import java.io.IOException;
+
+public class FileUtils {
+    public static byte[] readFile(String filePath) {
+        try {
+            var bytes = FileUtils.class.getResourceAsStream(filePath).readAllBytes();

```

Comment:
used getResourceAsStream() instead of FileInputStream because files are hosted outside working directory